test/onnx/test_onnx_common.py

```diff
@@ -38,6 +39,11 @@ def _run_model_test(test_suite: _TestONNXRuntime, *args, **kwargs):
     return verification.verify(*args, **kwargs)
 
 
+def parameterize_class_name(cls: Type, idx: int, input_dicts: Mapping[Any, Any]):
+    suffix = "_".join(f"{k}_{v}" for k, v in input_dicts.items())
```

Comment:
Do we need to use `parameterized.to_safe_name`?

Reply:
Didn't include it for now for simplicity. The method itself is unnecessarily a classmethod unfortunately. We could utilize it or do something similar, if someone needs to add parameters with nonconforming characters in the future.